hadoop-hdfs-project/hadoop-hdfs/src/main/resources/hdfs-default.xml

```diff
@@ -2006,12 +2006,12 @@
 
   <property>
     <name>dfs.namenode.fs-limits.max-xattr-size</name>
-    <value>13985</value>
+    <value>13755</value>
     <description>
       The maximum combined size of the name and value of an extended
-      attribute in bytes. The maximum allowed size is 13985, where the name
-      can take up to 255 bytes, and the value size can take up to 13730
-      bytes. If set to 0 the maximum allowed size (13985) is used.
+      attribute in bytes. The maximum allowed size is 13255 where the name
```

Comment:
typo in "The maximum allowed size is 13255 where the name...." it should be 13755

Reply:
fixed